tests/adapters_t/sites_t/test_moab.py

```diff
@@ -120,7 +124,7 @@ def tearDownClass(cls):
         cls.mock_config_patcher.stop()
         cls.mock_asyncssh_patcher.stop()
 
-    def mock_asyncssh_run(response, exit_status=0):
+    def mock_asyncssh_run(response, stderr="", exit_status=0):
```

Comment:
Either use one response object for example an `AttributeDict` or use consistent parameter names `response` -> `stdout`, please.